python/lsst/dax/apdb/apdb.py

```diff
@@ -71,7 +74,7 @@ def __enter__(self):
         self._timer1.start()
         return self
 
-    def __exit__(self, exc_type, exc_val, exc_tb):
+    def __exit__(self, exc_type: Optional[Type], exc_val: Any, exc_tb: Any) -> Any:
```

Comment:
Every method and argument needs to be annotated, otherwise `mypy` is not going to be happy. I guess it is not smart enough (yet) to know that `__exit__` has a defined signature. This could change in the future probably.